### ISSUE TITLE
Outdated or at least less relevant link in the aria-label page

### ISSUE THREAD

**Issue by iDad5** [mdn/content#33401]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

### What specific section or headline is this issue about?

The linked YouTube video

### What information was incorrect, unhelpful, or incomplete?

While I do agree with the general sentiment of cautioning about the use of aria-label in general, and — being dyslexic myself — truly sympathize with the person giving the speech, I think it should not be linked in the current way. The arguments in the linked video are mad from a mostly reasonable standpoint from about a decade ago. Screen-readers neglecting to implement proper support for Unicode and Browsers/Plugins displaying unsupported codepoints as squares instead of rendering them in a supporting system font is (or used to be) a shortcoming of their implementations and not a general problem with aria-label.

### What did you expect to see?

I agree that there should be cautioning about the use of aria-label, but the arguments in the video are either no longer valid, or have been based on shortcomings of vendor implementation. There are situations where — after continuous deliberation — the use of aria label in conjunction with e.g. the use of a Unicode based icon-font is a reasonable solution, and much better than neglecting an accessibility solution at all. 
In my opinion, an example like the eye icon inside (alongside) password-fields could be used as an example of such a compromise, but there might be betters too. Accessibility is very important, but I would argue that it is better to advocate 90%-solutions than to give the impression that designing an accessible website is all yes or no restrictions. 

### Do you have anything more you want to share?

The tenor of the video against icon-fonts in favor of SVG symbols is mostly opinionated and not really fact based.
